atlas/packages/components/src/components/VideoPreview/VideoPreview.tsx

```diff
@@ -1,47 +1,86 @@
-import React from "react"
+import React from "react";
 
-import { makeStyles, VideoPreviewStyleProps } from "./VideoPreview.styles"
-import Avatar from "./../Avatar"
+import { makeStyles, VideoPreviewStyleProps } from "./VideoPreview.styles";
+import Avatar from "./../Avatar";
 
 type VideoPreviewProps = {
-  title: string
-  channel?: string
-  channelImg?: string
-  showChannel?: boolean
-  poster?: string
-  onClick?: any
-  onChannelClick?: any
-} & VideoPreviewStyleProps
+	title: string;
+	channel?: string;
+	channelImg?: string;
+	showChannel?: boolean;
+	showMeta?: boolean;
+	time?: string;
+	views?: string;
+	poster?: string;
+	onClick?: any;
+	onChannelClick?: any;
+} & VideoPreviewStyleProps;
 
 export default function VideoPreview({
-  title,
-  channel,
-  channelImg,
-  showChannel = false,
-  poster,
-  onClick,
-  onChannelClick,
-  ...styleProps
+	title,
+	channel,
+	channelImg,
+	showChannel = true,
+	showMeta = true,
+	time,
+	views,
+	poster,
+	onClick,
+	onChannelClick,
+	...styleProps
 }: VideoPreviewProps) {
-  let styles = makeStyles({ showChannel, ...styleProps })
-  return (
-    <div css={styles.container} onClick={onClick}>
-      <div css={styles.coverContainer}>
-        <img css={styles.cover} src={poster} onClick={event => { event.stopPropagation(); onClick() }} />
-      </div>
-      <div css={styles.infoContainer}>
-        {showChannel && (
-          <div css={styles.avatar} onClick={event => { event.stopPropagation(); onChannelClick() }}>
-            <Avatar size="small" img={channelImg} />
-          </div>
-        )}
-        <div css={styles.textContainer}>
-          <h3 css={styles.title} onClick={event => { event.stopPropagation(); onClick() }}>{title}</h3>
-          {showChannel && (
-            <h3 css={styles.channel} onClick={event => { event.stopPropagation(); onChannelClick() }}>{channel}</h3>
-          )}
-        </div>
-      </div>
-    </div>
-  )
+	let styles = makeStyles({ showChannel, poster, ...styleProps });
+	return (
+		<div css={styles.container} onClick={onClick}>
+			<div css={styles.coverContainer}>
+				<div
```

Comment:
What's the reason behind this being a `div` instead of an `img`? I believe if an image is part of the content (as it is here) then it should be an `img` tag (for SEO, for screen readers, etc.)

Reply:
Good point, will change this to a regular `img` tag

Reply:
Done in 4177239 .

When we start adding stateful logic, we should add a `srcset` and serve different image sizes based on screensize.